hotwax/dxp-components
# Implemented: centralized facility switcher to be used in all app (#138)

### Related Issues

Closes: #138

### Short Description and Why It's Useful

Implemented centralized facility switcher component of the settings app in dxp-components so that it can be used in any of the app.

### Steps to verify
- Can verify the proper working of before/after emits using the inventory app PR: https://github.com/hotwax/inventory-count/pull/206
- can verify the proper implementation of this using receiving app PR: https://github.com/hotwax/receiving/pull/264

### Screenshots of Visual Changes before/after (If There Are Any)



**IMPORTANT NOTICE** - Remember to add changelog entry


### Contribution and Currently Important Rules Acceptance


- [x] I read and followed [contribution rules](https://github.com/hotwax/receiving#contribution-guideline)
